core/Columns/Dimension.php

```diff
@@ -171,40 +174,35 @@ public function getId()
      */
     public static function getAllDimensions()
     {
-        $dimensions = array();
+        $cacheId = CacheId::pluginAware('AllDimensions');
```

Comment:
We did not return dimensions that only extend the base `Dimension` class such as done by CustomVariables dimension. It now actually returns all dimensions